app/src/main/java/com/github/studydistractor/sdp/maps/MapsActivity.kt

```diff
@@ -83,7 +88,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         }
         marker?.tag = activity
         map?.setOnInfoWindowClickListener { activityMarker ->
-            val procActivity = activityMarker.tag as ProcrastinationActivity
+            val procActivity = activityMarker.tag as Distraction
             val intent = Intent(this, ProcrastinationActivityActivity::class.java)
```

Comment:
Since the Distraction Screen was added, I guess that this should take us to there instead of the (probably deprecated) ProcrastinationActivityActivity.

Reply:
I can't juste replace with screens, because MapsAcitivty is still not implemented screens.it is not straighforward for me to convert this. But once MapActivity is implemented with screens it will be straighforward to change. Or at least I don't know any way of doing it right now and I tried to do it

Reply:
Hmm maybe what @aizensoosuke proposed below will work? Because right now, this kind of breaks the functionality of maps, since it takes us to `ProcrastinationActivity`, which at this point is deprecated.